ContextDesktop/src/main/java/com/github/migi_1/Context/model/Environment.java

```diff
@@ -299,20 +295,32 @@ public void cleanup() {
     ////////////////////////////////////Below methods might be used later on when the pause screen is introduced.
     @Override
     public boolean isEnabled() {
-        // TODO Auto-generated method stub
         return false;
     }
 
     @Override
     public boolean isInitialized() {
-        // TODO Auto-generated method stub
         return false;
     }
 
     @Override
-    public void setEnabled(boolean arg0) {
-        // TODO Auto-generated method stub
+    public void setEnabled(boolean arg0) { }
 
+    /**
+     * Chooses a levelPiece from the randomly generated number rnd.
+     * @param rnd the random variable created by Math.random() (between 0 and 1).
+     */
+    private Spatial chooseLevelPiece(double rnd) {
+        if (rnd < 0.2) {
+            return assetManager.loadModel("Models/world1.j3o");
+        } else if (rnd < 0.4 && rnd > 0.2) {
+            return assetManager.loadModel("Models/world2.j3o");
+        } else if (rnd < 0.6 && rnd > 0.4) {
+            return assetManager.loadModel("Models/world3.j3o");
+        } else if (rnd < 0.8 && rnd > 0.6) {
+            return assetManager.loadModel("Models/world4.j3o");
+        } else {
+            return assetManager.loadModel("Models/world5.j3o");
+        }
```

Comment:
This can be done much nicer than with all these ugly if statements.
I'll explain it IRL.